examples/crowdfunding/src/lib.rs

```diff
@@ -75,6 +75,11 @@ pub trait Crowdfunding {
         }
     }
 
+    #[view(currentFunds)]
+    fn current_funds(&self) -> SCResult<BigUint> {
```

Comment:
there is special api for getBalance :). no need for such view func

Reply:
Forgot about that. But it's a good example to have it anyway.